x-pack/plugins/security_solution/server/usage/collector.ts

```diff
@@ -65,6 +66,163 @@ export const registerCollector: RegisterCollector = ({
+              },
+              alerts: {
+                type: 'long',
+                _meta: { description: 'Number of alerts generated by machine_learning rules' },
+              },
+              cases: {
+                type: 'long',
+                _meta: {
+                  description: 'Number of cases attached to machine_learning detection rule alerts',
+                },
+              },
+            },
+            threat_match: {
+              enabled: {
+                type: 'long',
+                _meta: { description: 'Number of threat_match rules enabled' },
+              },
+              disabled: {
+                type: 'long',
+                _meta: { description: 'Number of threat_match rules disabled' },
+              },
+              alerts: {
+                type: 'long',
+                _meta: { description: 'Number of alerts generated by threat_match rules' },
+              },
+              cases: {
+                type: 'long',
+                _meta: {
+                  description: 'Number of cases attached to threat_match detection rule alerts',
+                },
+              },
+            },
+            elastic_total: {
+              enabled: { type: 'long', _meta: { description: 'Number of elastic rules enabled' } },
+              disabled: {
+                type: 'long',
+                _meta: { description: 'Number of elastic rules disabled' },
+              },
+              alerts: {
+                type: 'long',
+                _meta: { description: 'Number of alerts generated by elastic rules' },
+              },
+              cases: {
+                type: 'long',
+                _meta: { description: 'Number of cases attached to elastic detection rule alerts' },
+              },
+            },
+            custom_total: {
+              enabled: { type: 'long', _meta: { description: 'Number of custom rules enabled' } },
+              disabled: { type: 'long', _meta: { description: 'Number of custom rules disabled' } },
+              alerts: {
+                type: 'long',
+                _meta: { description: 'Number of alerts generated by custom rules' },
+              },
+              cases: {
+                type: 'long',
+                _meta: { description: 'Number of cases attached to custom detection rule alerts' },
+              },
+            },
+          },
+          detection_rule_detail: {
+            type: 'array',
+            items: {
+              rule_name: {
+                type: 'keyword',
+                _meta: { description: 'The name of the detection rule' },
+              },
+              rule_id: {
+                type: 'keyword',
+                _meta: { description: 'The UUID id of the detection rule' },
+              },
+              rule_type: {
+                type: 'keyword',
+                _meta: { description: 'The type of detection rule. ie eql, query..' },
```

Comment:
Small nit: did you want `ie eql, query...` with three dots as an ellipsis?

Reply:
Thanks! Updated in https://github.com/elastic/kibana/pull/95659/commits/0c91a783675b0e48cd097b8d9b8af35919970cba